Dockerfile.dapper

```diff
@@ -3,21 +3,21 @@ FROM registry.suse.com/bci/golang:1.20
 ARG DAPPER_HOST_ARCH
 ENV ARCH $DAPPER_HOST_ARCH
 
-ENV HELM_VERSION v3.12.2
+ENV HELM_VERSION v3.13.3
 ENV HELM_UNITTEST_VERSION 0.3.2
 
 RUN zypper -n install git docker vim less file curl wget awk
 
 RUN curl -sL https://get.helm.sh/helm-${HELM_VERSION}-linux-${ARCH}.tar.gz | tar xvzf - -C /usr/local/bin --strip-components=1
 
 RUN if [ "${ARCH}" = "amd64" ]; then \
-        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.54.0; \
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.57.1; \
```

Comment:
Why was this upgraded?

Reply:
My usual reason of pulling in newer non-rancher components when backporting

Reply:
And I backed that out.